src/scss/components/_list.scss

```diff
@@ -99,9 +105,8 @@
       float: left;
     }
 
-    &::before {
-      content: '';
-      margin-right: 5px;
+    &:last-child {
```

Comment:
If we need to use `margin-top`, we need to reset the first child `margin-top`.

Reply:
It will not work that `margin-top` approach because of our fluid list.

Reply:
👍